scripts/core/ui/ui.js

```diff
@@ -1122,16 +1125,59 @@ function splitterWidget(superdesk, superdeskFlags, $timeout) {
                     }
 
                     authoring.width(authoringWidth / container.width() * 100 + '%');
+
+                    header.width(stage.outerWidth());
                 },
                 stop: function (e, ui) {
                     var container = ui.element.parent();
 
+                    var stage = ui.element.find('.stage.swimlane');
+                    var header = stage.find('.column-header.swimlane');
+
                     superdesk.monitoringWidth = workspace.outerWidth() / container.width() * 100 + '%';
                     superdesk.authoringWidth = authoring.outerWidth() / container.width() * 100 + '%';
 
+                    superdesk.headerWidth = superdesk.stageWidth = stage.outerWidth();
+
                     ui.element.css({
                         width: superdesk.monitoringWidth
                     });
+
+                    header.css({
+                        width: superdesk.headerWidth
+                    });
+                }
+            });
+        }
+    };
+}
+
+/*
+ * Header Resize directive for swimlane view, applies the same width of
+ * stage column to its fixed positioned header in order to resize it.
+ * sd-header-resize used in monitoring-view.html with respective sd-monitoring-group
+ *
+ */
+headerResizeDirective.$inject = [];
+function headerResizeDirective() {
+    return {
+        link: function(scope, elem) {
+            let resize = _.throttle(calcSize, 300);
+            function calcSize() {
+                let stageContainer = elem.find('.stage.swimlane');
+                let headerContainer = elem.find('.column-header.swimlane');
+                if (stageContainer && headerContainer) {
+                    if (headerContainer.width() !== stageContainer.width()) {
+                        scope.$applyAsync(function() {
+                            headerContainer.width(stageContainer.width());
+                        });
+                    }
+                }
+            }
+
+            scope.$watch(function() {
+                if (scope.viewColumn) {
+                    resize();
```

Comment:
this should be probably like `scope.$watch('viewColumn', resize);` otherwise it might call resize in every `$digest`